ci-scripts/yocto.groovy

```diff
@@ -166,7 +171,6 @@ void archiveImagesAndSDK(String yoctoDir, String suffix) {
 
         sh "rm -rf ${artifactDir}"
         sh "mkdir ${artifactDir}"
-
         // Copy images and SDK to the synced directory
         vagrant("/vagrant/ci-scripts/copy_to_archive ${yoctoDir}/build /vagrant/${artifactDir}")
 
```

Comment:
for the archiveArtifacts, add try/catch so we never get a null pointer exception.